website/docs/reference/widgets/text.md

```diff
@@ -1,94 +1,93 @@
-```
 
-When you tick the checkbox, it will enable the Disable link property, and the input will be parsed as text.
+| Property            | Description                                                                                                                                                                                |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **Text**            | Sets the text to be displayed.                                                                                                                                                             |
+| **Scroll**   | This option enables scrolling within the boundaries of the Text widget. It allows you to display longer text within a small area of the app without truncating it, and lets users scroll through the content to view it in its entirety.                             |
+| **Truncate text**   | This feature truncates the text in the Text widget and adds three ellipses at the bottom left corner. If the text is longer than the widget area, clicking on the ellipses opens up a pop-up that displays the entire text.                                                                                                                           |
+| **Visible**         | It controls the widget's visibility on the page. When turned off, the widget would not be visible when the app is published.                                                                |
+| **Animate Loading** | Controls the widget's loading animation. This can be controlled with JS until all the widgets are rendered. |
+| **Disable link**    | It parses any link in the widget as standard text.                                                                                                                                         |
+| [**Height**](/reference/widgets/#height)        | It configures how a widget’s height reacts to content changes. It has three possible configurations:<br/>**Fixed**: The height of the widget remains as set using drag and resize.<br/>**Auto Height**: The height of the widget reacts to content changes.<br/>  **Auto Height with limits**: Same as Auto height, with a configurable option to set the minimum and maximum number of rows that can be occupied by the widget.                                      |
 
-<VideoEmbed host="youtube" videoId="bmPk0arvZQM" title="Disable link" caption="Disable link"/>
 
 
-### Binding Properties
 
-These properties help you share values between widgets and also allow you to easily access the widget property within Queries or JS functions.
+### Reference properties
+These properties can be referenced in other widgets, queries, or JS functions using the dot operator.
 
 | Property      | Description                                                   | Code Snippet         |
```

Comment:
remove code snippet column. give example in the line 

These properties can be referenced in other widgets, queries, or JS functions using the dot operator, for example `textName.isVisible`